client/js/ace/snippets/fbp.js

```diff
@@ -0,0 +1,669 @@
+define("ace/snippets/fbp",["require","exports","module"], function(require,exports,module){
```

Comment:
This is totally good! :)

The next steps are creating this dynamically when we are starting the server in server/app.js (Where everything starts)